tests/unit/utils.ts

```diff
@@ -121,9 +125,24 @@ export function generateCompletionData(
     "vpython"
   );
 
-  console.log(`Input .tst script: \n ${textDocument.getText()} \n`);
-
-  return getTstCompletionData(textDocument, completion);
+  // If Coded test
+  if (cppTest.cppNode && cppTest.lineSoFar) {
+    console.log(`Input .cpp script: \n ${textDocument.getText()} \n`);
+    const enviroPath = getEnviroNameFromTestScript(tstFilepath);
+    if (enviroPath) {
+      return getCodedTestCompletionData(
+        cppTest.lineSoFar,
+        completion,
+        enviroPath
+      );
+    }
+    throw new ReferenceError("enviroPath is undefined.");
+
+    // TST test
```

Comment:
Bad comment

Reply:
Changed comment in 9038de54b41c65eec73f8f620cbae868eb5a3936 .